creature.cpp

```diff
@@ -1381,14 +1392,31 @@ CreatureAction Creature::fire(Vec2 direction) const {
   });
 }
 
+static bool hasBridge(const Position& pos) {
+  if (auto floor = pos.getFurniture(FurnitureLayer::GROUND))
+    if (floor->getType() == FurnitureType::BRIDGE)
```

Comment:
Btw, please avoid directly referencing FurnitureType elements because it's easy to break this code when adding new furniture that has the same logic.